spec/bolt/cli_spec.rb

```diff
@@ -582,6 +582,45 @@ def stub_config(file_content = nil, base = nil)
             ]
           )
         end
+
+        it "shows an individual task data in json" do
+          task_name = 'sample::params'
+          options = {
+            mode: 'task',
+            action: 'show',
+            object: task_name
+          }
+          cli.execute(options)
+          expect(JSON.parse(@output.string)).to eq(
+            "name" => "sample::params",
+            "description" => "Task with parameters",
+            "executable" => "#{Dir.pwd}/spec/fixtures/modules/sample/tasks/params.sh",
```

Comment:
This seems fragile. Is it better to remove the executable kv pair from the object here, remove it when we get the task info from the compiler, or to just leave it as is?

Reply:
Lets just not test that part of the output. I don't know how important showing the executable path is.